types/react-elm-components/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "private": true,
+    "name": "@types/react-elm-components",
+    "version": "1.0.9999",
+    "projects": [
+        "https://github.com/evancz/react-elm"
+    ],
+    "dependencies": {
+        "@types/react": "^18"
```

Comment:
Are you definitely wanting to pin this? Or is `*` ("anything") workable? Most DT packages do not pin these for maximum compat.

Reply:
You're right. No need to pin this!